### DISCUSSION TITLE
What is meant by reads and writes in the process view?

### DISCUSSION THREAD

**Discussion by ElSamhaa** [ClementTsang/bottom#1540] [Q&A] [answered]
Is it disk IO per process?
If that is correct, is it possible to add network IO per process?

**Comment by ClementTsang** [comment, marked as answer]
It's generally disk IO, though the crate I use apparently returns total IO for Windows only for whatever reason. I'll note that down somewhere, sorry for the confusion.

As for network IO at a per-process level, that would be nice to add, though I have no idea where to look for that stat across all operating systems so I don't know how easy/hard it would be to implement.
